src/components/Documentation/Markdown/Main/styles.module.css

```diff
@@ -70,6 +70,10 @@
       }
     }
 
+    a.token.args {
+      color: var(--color-light-blue);
```

Comment:
A `a`'s `outline-color` is `--color-gray-hover` by default which isn't viewable on dark backgrounds. Adding `outline-color: var(--color-light-blue-hover);` here should allow keyboard users to see when these links are being focused on.